script/Ciel/Day 13/13_01_01_CIEL13_2.txt

```diff
@@ -467,55 +494,56 @@ I am truly broken.
 [sha:92dea0e135af9e7c61b2aa60da10d99f0aa82cd2]{
 -- Page 97, Offset 33436.
 -- 俺は、駄目だ。
-I'm hopeless.
+It's over for me.
 }
 [sha:9c88a98fb3e8df2d84439d45be6ca9313e367ec4]{
 -- Page 97, Offset 33437.
 -- これ以上、正気でいられる自信がない。
-I can't retain my sanity for any longer.
+I don't know how much longer I can stay sane.
 }
 [sha:3882e54e20a25540b42fbf55a5a5861473f5c821]{
 -- Page 98, Offset 33438.
 -- 「あはは、あはははは」
-"Ahaha, ahahahaha."
+"%{i}Ahaha%{/i}, %{i}ahahahaha%{/i}."
 }
-[sha:f8d9926ea8d6656dbcb3df9e926ad261bff60181]{
+[offset:33439]{
 -- Page 98, Offset 33439.
 -- 外。
-Outside.
+Out.
 }
 [sha:cbe22c87c5ed758674b28c0a43b19735499944c9]{
 -- Page 98, Offset 33440.
 -- 外に出ないと。
-I need to go outside.
+I need to get out.
 }
 [sha:482b3ce31d8c49e91f50486be0b1e170a014b958]{
 -- Page 98, Offset 33441.
 -- このままだと生き返った■■さんと鉢合わせる。
-If I stay here, I'll end up running into the revived ■■-san.
+If I stay here, I'll end up running into the reanimated ■■■-san.
 }
 [sha:e2d068ad458d9f51db8da78af9d19aa1d0b26488]{
 -- Page 98, Offset 33442.
 -- もうそうなったら自我を保てる筈がない。
-If that happens, then there's no way my ego will remain intact.
+A shock like that would be enough to shatter what remains of my sense of self.
```

Comment:
I think "persona" might be more appropriate for this 自我, but it's sort of a esoteric point. Essentially, his 自我 is now the thing he wears on the outside, the true him is being overwritten by Roa. While he still acts as Shiki, his persona is preserved, but it's about to crumble.

Anyway, it might not really matter.